src/mcpil.py

```diff
@@ -534,6 +534,11 @@ def main():
     window.title('MCPIL')
     window.geometry('512x400')
     window.resizable(True, True)
+    
+        if not time_now.day == 1 and not time_now.month == 4: #The month and the day isn't the Aprils's fools day
```

Comment:
Is this indentation intentional? Also, the code might be slightly clearer if you swap the clauses, so instead of ``if not time_now.day == 1 and not time_now.month == 4:`` it is ``if time_now.day == 1 and time_now.month == 4:``, just my opinion though.

Reply:
idk, just do something if needed, good night

Reply:
> idk, just do something if needed, good night

You need to fix it yourself